lib/filter/filter-pipe.c

```diff
@@ -67,25 +66,24 @@ log_filter_pipe_queue(LogPipe *s, LogMessage *msg, const LogPathOptions *path_op
 
   res = filter_expr_eval_root(self->expr, &msg, path_options);
 
+  msg_trace("<<<<<< filter rule evaluation result",
+            evt_tag_str("result", res ? "matched" : "unmatched"),
```

Comment:
[optional]

Is there a reason we simplify the `result` string from `MATCH - Forwarding message to the next LogPipe` to `matched` and similarly with `unmatched`? Could we keep it like before?

Reply:
We could, but I never liked those versions :) I could separate that out.

but the patch already changes where the result is reported, which makes it a lot easier to follow how filters/parsers are applied (at least for me as the >>> / <<< lines nicely encapsulate the checks that a specific filter performs)

Reply:
btw, the "Forwarding message to the next..." text actually was printed after the forwarding already happened and we got back from the very end of the pipeline. I just feel those were very verbose.